### ISSUE TITLE
ECONNRESET when trying to create an index

### ISSUE THREAD

**Issue by mjschranz** [phillro/node-elasticsearch-client#74]
Perhaps this is the wrong place but I was looking for some help solving this problem while using this node library.

At this point I'm initializing my client with the default values elasticsearch normally uses so

``` javascript
options = {
  port: 9200,
  host: "localhost",
  pathPrefix: "test",
  secure: true
};
```

Now switching the secure flag to being a false value doesn't throw an error but does leave me with `No handler found for uri [test/Test_Makes] and method [PUT] data`. Frankly I have no idea what this means.

Basically my question(s) is/are, what does the secure flag here do when initializing my ES client? What does not having a handler mean there?

Again, sorry for polluting the issues with this question set. I tried scouring through the ElasticSearch API to make connections between this wrapper library and their API but alas came up empty handed.

Many Thanks.
